src/components/atoms/AppCheckboxInput.vue

```diff
@@ -6,5 +6,39 @@ const props = defineProps({
 </script>
 
 <template>
-  <input type="checkbox" :disabled="props.disabled" :checked="props.checked" />
+  <input class="form-check-input" type="checkbox" :disabled="props.disabled" :checked="props.checked" />
 </template>
+
+<style>
+.form-check-input[type='checkbox'] {
```

Comment:
```suggestion
.form-check-input {
```
Since you defined a specific class for this input, you don't need this selector